src/nodes.coffee

```diff
@@ -404,6 +404,14 @@ exports.InfinityLiteral = class InfinityLiteral extends NumberLiteral
   compileNode: ->
     [@makeCode '2e308']
 
+exports.NaNLiteral = class NaNLiteral extends NumberLiteral
+  constructor: ->
+    super 'NaN'
+
+  compileNode: (o) ->
+    code = [@makeCode '0/0']
+    if o.level >= LEVEL_OP then @wrapInBraces code else code
```

Comment:
I settled on using `@makeCode` instead of `new Op` since that allows `NaN` to be rendered as `0/0` instead of `0 / 0`.